src/shared/utils.ts

```diff
@@ -552,9 +553,16 @@ export function pictrsDeleteToast(
       duration: 10000,
       onClick: () => {
         if (toast) {
-          window.location.replace(deleteUrl);
-          alert(deletePictureText);
-          toast.hideToast();
+          fetch(deleteUrl, {})
```

Comment:
Why are you doing a fetch here, doesn't it delete the picture elsewhere? This should just be the toast.

Reply:
No, it doesn't here we had a simple window.location() which the browser does not like.

Reply:
In that case use a callback function for onClick, because this should really just have toast-related code.

Reply:
I don't really understand the difference, can you make the change or a proposal?

Reply:
```ts
export function pictrsDeleteToast(...
 onClick: () => void
) {
...
  onClick: () => {
    toast.hideToast();
    onClick();
  }
}
...
pictrsDeleteToast(.... () => { do fetching stuff here }
```

Reply:
I think you misunderstand the logic here. The picture is deleted when the user clicks on the toast message itself. So it must be inside onClick?! Please just fix it yourself, I'm losing interest like this. It's not worth it for such small but annoying bug. :(